packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1183,7 +1184,12 @@ function build(
     );
 
     // here we finally invoke the vat code, and get back the root object
-    const rootObject = buildRootObject(harden(vpow), harden(vatParameters));
+    baggage = collectionManager.provideBaggage();
```

Comment:
I deliberately put the creation of the baggage immediately before the invocation of `buildRootObject` so that I could be sure, without having to think too deeply about it, that the environment was properly set up (as it would have to be since you need to be able to create collections inside `buildRootObject`).  This was mostly a labor saving move on my part, since the choice of where to do this seemed otherwise relatively arbitrary.  A quick check says moving it to where you suggest shouldn't be a problem, aside from the inevitable rejiggering of some of the tests due to the resulting change in order of DB accesses.